templates/example.html

```diff
@@ -13,9 +13,12 @@
   <body>
     <div class="body">
       {{> header.html}}
-      {{#sections}}
+      {{#data.sections}}
```

Comment:
@juliantoledo understood, but couldn't we just add the function as a field in the args object?